### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by danylofedas**

### CODE REVIEW

**Review by danylofedas — COMMENTED** [review]
1

**Review by Roxyprog — CHANGES_REQUESTED** [review]
Make all comments in English. And this code must be in bmi.py

File: console/bmi.py [hunk 1]
```diff
@@ -9,10 +11,36 @@
 
 #It is necessary to add recommendations according to the result of BMI
 if bmi_value < 18.5:				#Analysis of BMI
-        print("Недостатня вага")
```
Comment (Roxyprog) [inline console/bmi.py:12]: This row is unnecessary/

File: console/bmi.py [hunk 2]
```diff
@@ -9,10 +11,36 @@
 
 #It is necessary to add recommendations according to the result of BMI
 if bmi_value < 18.5:				#Analysis of BMI
-        print("Недостатня вага")		
+        print("Недостатня вага")
```
Comment (Roxyprog) [inline console/bmi.py:14]: This row is unnecessary.

File: console/bmi.py [hunk 3]
```diff
@@ -9,10 +11,36 @@
...
+	# Random recommendations for underweight
+	recommendations_underweight = [
+        "Ваш ІМТ є нижчим норми. Перед початком тренувань рекомендуємо звернутися до спеціаліста. Ви можете обрати план тренувань «набір ваги» і вводити у раціон більшу кількість калорій.",
+        "Ваш ІМТ є заниженим. Перед початком тренувань рекомендуємо звернутися до спеціаліста для індивідуальної консультації.",
+        "Ваш ІМТ є нижчим від норми. Рекомендуємо звернутися до спеціаліста для індивідуальної консультації."
+    ]
+    print(random.choice(recommendations_underweight))
 elif bmi_value >= 18.5 and bmi_value < 24.9:
         print("Нормальна вага")
```
Comment (Roxyprog) [inline console/bmi.py:23]: This row is unnecessary/

File: console/bmi.py [hunk 4]
```diff
@@ -9,10 +11,36 @@
...
+        # Random recommendations for normal weight
+    recommendations_normal = [
+        "Вітаємо, у вас ІМТ в межах норми. Ми рекомендуємо обрати наш план тренувань «підтримка ваги».",
+        "Ваш ІМТ в нормі. Рекомендуємо план «підтримка ваги» для підтримки вашого стану.",
+        "Вітаємо, у вас нормальний ІМТ. Ми рекомендуємо план тренувань «підтримка ваги»."
+    ]
+    print(random.choice(recommendations_normal))
 elif bmi_value >=25 and bmi_value  < 29.9:
         print("Надмірна вага")
```
Comment (Roxyprog) [inline console/bmi.py:32]: This row is unnecessary/

File: console/bmi.py [hunk 5]
```diff
@@ -9,10 +11,36 @@
...
+        recommendations_overweight = [
+        "Ваш ІМТ є завищеним, рекомендуємо план тренувань «зниження ваги».",
+        "Ваш ІМТ трохи перевищує норму. Рекомендуємо вам план тренувань «зниження ваги».",
+        "Ваш ІМТ є трохи завищеним. Рекомендуємо вам план тренувань «зниження ваги»."
+    ]
+        print(random.choice(recommendations_overweight))
 else: 
         print("Ожиріння")
```
Comment (Roxyprog) [inline console/bmi.py:40]: This row is unnecessary.

**Review by Roxyprog — CHANGES_REQUESTED** [review]

**Review by Roxyprog — APPROVED** [review]
Everything is ok)

**Pull request merged**